### PR TITLE
Add ACRE setting to change initial direct speech volume

### PR DESCRIPTION
**Pull request by 3Mydlo3**
**When merged this pull request will:**
- title

### CODE REVIEW

File: addons/acre/functions/fnc_adjustVoiceVolume.sqf [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ *
```
Comment (veteran29) [inline addons/acre/functions/fnc_adjustVoiceVolume.sqf:4]:
```suggestion
```

File: addons/acre/functions/fnc_adjustVoiceVolume.sqf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * None
+ *
+ * Public: No
+ */
+
+if (!hasInterface) exitWith { nil };
+
+private _volumeLevel = [GVAR(startBppVolume), GVAR(startLeaderVolume)] select (player isEqualTo leader player);
+
+private _volume = switch (round _volumeLevel) do {
```
Comment (veteran29) [inline addons/acre/functions/fnc_adjustVoiceVolume.sqf:19]: What do you think about making this setting a dropdown instead of rounding the slider number?

Comment (3Mydlo3) [reply addons/acre/functions/fnc_adjustVoiceVolume.sqf:19]: I thought about it when doing that but I could not find any translations for speech volume levels in ACRE. But yes, we can do it ourself and this should be more user friendly.

**Comment by veteran29** [comment]
Temporary solution until it will be in workshop released ACRE version, see: https://github.com/IDI-Systems/acre2/issues/1077
(we might consider creating PR to ACRE too)

**Pull request merged**